presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorMetadata.java

```diff
@@ -521,6 +522,15 @@ default void dropView(ConnectorSession session, SchemaTableName viewName)
         return Optional.empty();
     }
 
+    /**
+     * Gets the schema definition for the specified schema.
+     */
+    default Optional<ConnectorSchemaDefinition> getSchema(ConnectorSession session, CatalogSchemaName schemaName)
+    {
+        // Only the Hive Connector supports this. This default simplifies all other connectors.
+        return Optional.of(new ConnectorSchemaDefinition(Collections.emptyMap(), Optional.empty()));
```

Comment:
See the comment... With this we can avoid adding a new method to each and every connector.